### PR TITLE
EC-2619 Ibm deploy fix for redis connection(s)

### PR DESCRIPTION
**Pull request by lbiedinger**

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -31,9 +31,19 @@ class Application < Rails::Application
       # Setup redis as the cache_store.
       # This is required for sidekiq, which uses redis to queue jobs.
       config.cache_store = begin
-        redis_config = Rails.application.config_for(:redis).symbolize_keys
+        redis_config = Rails.application.config_for(:redis).deep_symbolize_keys
+        opts = {}
+        if redis_config[:ssl_params]
+          opts.merge!({
+                        ssl: :true,
```
Comment (rwd) [inline config/application.rb]: Should this be `ssl: true`?

File: config/application.rb [hunk 2]
```diff
@@ -31,9 +31,19 @@ class Application < Rails::Application
...
+                        scheme: 'rediss',
```
Comment (rwd) [inline config/application.rb]: Is this necessary given that it's in the URL too?

File: config/initializers/sidekiq.rb [hunk 3]
```diff
@@ -1,15 +1,16 @@
 # frozen_string_literal: true
 
-redis_url = nil
+redis_options = nil
 
 if Rails.cache.is_a?(ActiveSupport::Cache::RedisStore)
-  redis_url = Rails.cache.data.connection[:id]
+  redis_options = Rails.cache.data.instance_values['options']
+  redis_options.merge!(namespace: 'sidekiq' )
```
Comment (rwd) [inline config/initializers/sidekiq.rb]: Remove the trailing space before the closing parenthesis.

File: config/application.rb (hunk 2 above)
Comment (lbiedinger) [reply config/application.rb]: yeah, whatever way the cache_store is initialized actually takes apart the URL into the individual parts and by the time it comes to making the connection the scheme is lost unless it's explicitly passed here.

File: config/application.rb (hunk 1 above)
Comment (lbiedinger) [reply config/application.rb]: it's taken straight from here: https://github.com/redis/redis-rb#ssltls-support
I hadn't tried a boolean value.

**Pull request merged**